src/components/PeoplePage/Parent/Parent.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from 'react';
+import { Person } from '../../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+import { ParentProps } from '../PeoplePage.types';
+
+function getParentByName(name: string | null, people: Person[]) {
+  const parent = people.find(person => person.name === name);
+
+  return parent || undefined;
```

Comment:
But it's already undefined if nothing is found
I think will be more logical to set null, but it's not critical